### ISSUE TITLE
廣告

### ISSUE THREAD

**Issue by a0989006421** [a0989006421/001#21]
[Adblock Plus]
! Title: LowTechFilter Pureview News Mobile
! Version: 2026.0124.1
! Expires: 1 hour
! Homepage: https://t.me/AdBlock_TW
! ----------------------------------------------------------------------
!  三立新聞
www.setn.com##div[class="newsflashBox"]
www.setn.com##div[class="right-area"]
www.setn.com##div[id="coniBox"]
www.setn.com##div[id="donateBtn"]
www.setn.com##div[class="hotnav-bar"]
www.setn.com##div[class="readmoreCont"]
www.setn.com##div[class="tagNewsArea"]
www.setn.com##div[class="newsTextADBox"]
www.setn.com##div[class="extendnews"]
www.setn.com##div[class="ai_area_m"]
www.setn.com##.page-float
www.setn.com##div[id="scroll"]
www.setn.com##div[class="page-pre-area"]
www.setn.com##div[class="page-next-area"]
www.setn.com##.livevodBox setn.com###LiveVOD
||www.setn.com/VOD.aspx?IsLive=true
www.setn.com##div[class="video-container"]
www.setn.com##.news-all-area:has(> .hotnews-title-area)
www.setn.com##.news-all-area:has(> .live-title-area)
www.setn.com##.news-all-area:has(> .page-pre-area)
www.setn.com##div[class="hotnews-title-area"]
www.setn.com##div[id="KeywordCloud"]
www.setn.com##div[data-layid="ChannelArea"]
www.setn.com##div[class="channelBox"]
www.setn.com##div[class="social-media-icon"]
www.setn.com##div[class="video-container livevodBox"]
www.setn.com##.newsTextADBox
www.setn.com##.news-area-ad
www.setn.com##.page-project-area
www.setn.com##.ad
www.setn.com##div[id="KeywordCloud"]

! 自由時報
news.ltn.com.tw#$#div[class="headerP_high"] { height: 100px !important;}
news.ltn.com.tw##div[class="caption"]
news.ltn.com.tw##div[class="related boxTitle"]
news.ltn.com.tw##div[id="todayHotNews"]
news.ltn.com.tw##div[class="see_more boxTitle"]
news.ltn.com.tw##p[class="appE1121"]
news.ltn.com.tw##div[id="marqueeHeader"]

! Yahoo 新聞
tw.news.yahoo.com##div[id="module-view-comments"]
tw.news.yahoo.com##p[class="read-more-vendor"]
tw.news.yahoo.com##p[class="read-more-editor"]
||tw.news.yahoo.com^$cookie=*;maxAge=10;sameSite=lax
tw.news.yahoo.com##ul[class^="caas-list"]
tw.news.yahoo.com##a:matches-attr("data-i13n")
tw.news.yahoo.com##div[id="module-relatedStories"]
tw.news.yahoo.com##div[id="module-originalContents"]

! 旺旺中時
!#if (adguard && adguard_app_android)
||static.chinatimes.com^$replace=/\.\.\/\.\.\/images\/2020\/logo-chinatimes2020\.svg/https\:\/\/filter\-assets\.futa\.gg\/ct02\.svg/
||static.chinatimes.com^$replace=/\.\.\/\.\.\/images\/2020\/logo-chinatimes-alt2020\.svg/https\:\/\/filter\-assets\.futa\.gg\/ct02-PCview\.svg/
!#endif
chinatimes.com#$#div[class="main-header-logo"] { top: 16px !important; }

! 風傳媒
www.storm.mg##div[class="article_ads_p_wrapper"]
www.storm.mg##div[id="premium_block"]
www.storm.mg##div[id="author_inner"]
www.storm.mg##div[id="related_wrapper"]
www.storm.mg##div[id="news_list_wrapper"]
www.storm.mg##div[id="category_content"]
www.storm.mg##div[id^="dablewidget_"]
www.storm.mg##div[id="more_news_wrapper"]
www.storm.mg##div[id="footer_links_wrapper"]
www.storm.mg##div[id="app_store_wrapper"]
www.storm.mg#$#div[id="article_content"] { margin-right: unset !important; }
www.storm.mg#$#div[id="footer_inner"] { padding-bottom: 10px !important; padding-top: 10px !important}
www.storm.mg##a[href="/support-us"]
www.storm.mg##div[id="donation_block"]
www.storm.mg##a[class$="tag_vip"]
www.storm.mg##a[class="nav_btn event"]
www.storm.mg##a[id="prevPage_btn"]
www.storm.mg##div[id="header_btn_wrapper"]

! UDN
udn.com##[class="udn--event__holder"]
udn.com##[class="orange-promotion"]
udn.com##[class^="breaking-news__wrapper"]
udn.com##[class="vip-edn--promotion"]
udn.com##[class="udn-channel--promotion"]
udn.com##[class^="feature-guess"]
udn.com##[class^="mobile-recommendation"]
udn.com##[class="newsletter"]
udn.com##[class="footer-social"]
udn.com##[class="footer"]
udn.com##[class^="related-news"]
udn.com##[class^="more-news"]
udn.com##[class^="facebook-comment"]
udn.com##[class^="discuss-board"]

! UDN Money
money.udn.com##div[class="breaking-news-container "]
money.udn.com##section[class="article-switch"]
money.udn.com##div[class="story-flex-bt-wrapper"]
money.udn.com##section[class="article-comments"]
money.udn.com##section[class="sitemap toggle-list"]
money.udn.com##a[id="goTop"]
money.udn.com##div[class^="edn-ads"]

! UDN pets
pets.udn.com##div[class="story variety extend"]
pets.udn.com##div[class="story-list__header"]
pets.udn.com##div[id="comments"]
pets.udn.com#$#section[class="article-content__wrapper"] { width: unset !important; }
pets.udn.com#$#.article-content__cover picture { height: unset !important; width: unset !important; }

! CNA
cna.com.tw##[id="jsAppInfo"]
cna.com.tw##[class="appInfo"]
cna.com.tw##[class^="Subscription"]
cna.com.tw##[class="articlekeywordGroup"]
cna.com.tw##[class*="moreArticle"]
cna.com.tw##[class="paragraph BtnShareGroup appDownload"]
cna.com.tw##[class*="social-group"]
cna.com.tw##[class*="advertiseBox"]
cna.com.tw##[class^="privacy-bar"]
cna.com.tw##footer

! Dcard
dcard.tw##div[class^="atm"]:has(> div[id^="div-gpt-ad"])
dcard.tw#?#button[-ext-contains="/app 內打開/i"]
dcard.tw##div[class]:has(> button[class]:has-text(/用 App 查看完整內容/))

! technews 科技新報
technews.tw##div[class="tipbar"]
technews.tw##div[id="secondary"]
technews.tw##div[class="googlenews_Content"]
technews.tw##div[class="newsLetter_articleContent newsLetter_articleColumn"]
technews.tw##nav[class="nav-single"]
technews.tw##div[id="fbSEOComments"]
technews.tw##a[id="scrolldown"]
technews.tw##div[class="footer_frame"]
||technews.tw/wp-admin/admin-ajax.php*

! 商周
businessweekly.com.tw##div[class^="Single-right-part"]
businessweekly.com.tw##section[class^="Post-section"]
businessweekly.com.tw##section[class^="Promotion"]
businessweekly.com.tw##div[class="promote_subscribe"]
businessweekly.com.tw##div[class="cover_summary_story"]
businessweekly.com.tw##section[class^="Single-related"]
businessweekly.com.tw##section[class="d-xs-none d-md-block"]
businessweekly.com.tw##section[class^="Single-section"]
businessweekly.com.tw##button[data-label="訂閱"]
businessweekly.com.tw##div[class^="Links"]
businessweekly.com.tw##div[class^="Border"]
businessweekly.com.tw##div img[class="Logo-white lazy"]
businessweekly.com.tw##div[class^="Logos"]
businessweekly.com.tw##section[class^="Single-sidebar"]
businessweekly.com.tw##body .Google-special:not(#style_important)
businessweekly.com.tw#$#.col-lg-8 { flex: unset !important; max-width: unset !important}
businessweekly.com.tw#$#.col-md-7 { flex: unset !important; max-width: unset !important}
||ibw.bwnet.com.tw/file/js/vendor/infinite-scroll.pkgd.min.js

! CTwant
ctwant.com##div[class="m-adpopup"]
ctwant.com##div[class^="l-ad__wrapper"]
ctwant.com##div[class^="l-ad__article"]
ctwant.com##div[class="l-series"]

! NowNews
nownews.com##div[class="subsitetoolsBlk"]
nownews.com##h3[class="hotnews-top"]
nownews.com##section[class="breakingNews sticky"]
nownews.com##section[class="breakingNews"]
nownews.com##div[class="custom-blk"]
nownews.com##div[class="related-item"]
nownews.com##div.keywordBlk 
nownews.com##section.voteBlk
nownews.com##div[class="nnBlk immediate mb-1"]
nownews.com##ul[class="social-media"]
nownews.com##div[class="event-holder"]

! 世界新聞網
worldjournal.com##section[class="breaking-news__wrapper"]
worldjournal.com##div[class$="inline-ads"]
worldjournal.com##section[class="next-page"]
worldjournal.com##div[class^="cate-list__container"]
worldjournal.com##section[class$="facebook-comment"]
worldjournal.com##section[class="footer-social"]

! Ettoday
ettoday.net##div[id="manual_links"]
ettoday.net##div[id="recomm-news"]
ettoday.net##div[class^="etvideo_collect_2"]
ettoday.net##div[class="block block_1"]
ettoday.net##div[class="block block_2"]
ettoday.net##div[id="etnews_live_player"]
ettoday.net##div[class="nav_3"]
ettoday.net##a[class="download_app"]
ettoday.net#%#//scriptlet('set-cookie-reload', 'liveShowClose', '1', '/', 'ettoday.net')
ettoday.net##div[id="hot-news"]
ettoday.net##div[class="part_area_1 margin"]
ettoday.net##div[class="part_area_1"]
ettoday.net##div[class="text_ticker_1"]:remove()
ettoday.net##div[class="gototop"]

! 美麗佳人
marieclaire.com.tw##p[class^="extendArticle"]
marieclaire.com.tw##ul[class^="extendArticle"]
marieclaire.com.tw##div[class="adArticlebottom"]
marieclaire.com.tw##div[class="extendRead"]
marieclaire.com.tw##div[class="magazine"]
marieclaire.com.tw##p[class="subscribeTitle"]
marieclaire.com.tw##div[class="subscribeBox"]

! 電影神搜
news.agentm.tw##div[id="agmn_a_beginArticle_M"]
news.agentm.tw##div[class="relative"]
news.agentm.tw##aside[class="pside"]
news.agentm.tw##aside[class="aside"]
news.agentm.tw##footer[id="footer"]

! CTWant
ctwant.com##div[class="l-header__downloadapp"]
ctwant.com##div[id="article-tag-bottom"]
ctwant.com##div[class="l-section"]
ctwant.com##aside[class="l-sidebar-mobile"]
ctwant.com##div[class="l-footer__social"]

! 鏡傳媒 Mirrormedia
www.mirrormedia.mg##div[class^="gpt-placeholder__Container"]
www.mirrormedia.mg##section[class^="related-article-list"]
www.mirrormedia.mg##div[class^="support-mirrormedia-banner__Container"]
www.mirrormedia.mg##div[class^="social-network-service__Wrapper"]
www.mirrormedia.mg##section[class^="normal__StoryEnd"]
www.mirrormedia.mg#$#article { margin: unset !important; max-width: unset !important; }
||www.mirrormedia.mg^$cookie=/AviviD.+/

! 壹蘋新聞
tw.nextapple.com##a[class="app-m"]
tw.nextapple.com##aside[id="side-bar"]
tw.nextapple.com##div[class="post-related partner"]
tw.nextapple.com##div[class="post-comments"]
tw.nextapple.com##div[class="tags"]
tw.nextapple.com#$#div[id="main-content"] { width: 100% !important;}
tw.nextapple.com##a:has-text(/推薦新聞/)
tw.nextapple.com##a[href^="https://reporting.nextapple.com"]
tw.nextapple.com##hr
tw.nextapple.com##div[class="scrollTool"]
tw.nextapple.com##div#top-slider
tw.nextapple.com##div#marquee

# 遠見雜誌
gvm.com.tw##section[class^="newsletter"]
gvm.com.tw##div[class^="article-head_marketing"]
gvm.com.tw##div[class="media-sec"]
gvm.com.tw##div[class="rights-sec"]
gvm.com.tw##div[id^="article_end_"]
gvm.com.tw##div[data-recommend_placement="article_float_1"]
gvm.com.tw##div[id="content_area"]
gvm.com.tw##a[class="list-cards"]
gvm.com.tw##div[data-recommend_placement="article_1_bottom"]
gvm.com.tw##div[class="show-small"]
||rec.gvm.com.tw/gvm_recommend.php^
||gvm.com.tw/article/next_article^
||gvm.com.tw/recommend/article^

# 賴傳媒
lai-media.net##div[class="page-right"]:remove()
lai-media.net##div[class="marqueebar"]
lai-media.net##.header_btm:remove()
lai-media.net##section[class="sec_more_article"]
